src/form-elements/FormElementBoolean.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react'
+import clsx from 'clsx'
+import useBooleanState from '../hooks/useBooleanState'
+import { FormTypes } from '@oneblink/types'
+import FormElementLabelContainer from '../components/FormElementLabelContainer'
+import { Switch } from '@material-ui/core'
+
+type Props = {
+  id: string
+  element: FormTypes.BooleanElement
+  value: unknown
+  onChange: FormElementValueChangeHandler<boolean>
+  displayValidationMessage: boolean
+  validationMessage: string | undefined
+}
+
+function FormElementBoolean({
+  id,
+  element,
+  value,
+  onChange,
+  validationMessage,
+  displayValidationMessage,
+}: Props) {
+  const [isDirty, setIsDirty] = useBooleanState(false)
+  return (
+    <div className="cypress-boolean-element">
+      <FormElementLabelContainer
+        className="ob-boolean"
+        id={id}
+        element={element}
+        required={element.required}
+        leading={
+          <Switch
+            id={id}
+            name={element.name}
+            color="default"
+            className={clsx('ob-boolean__input', 'cypress-boolean-control', {
+              'ob-boolean__input-checked': !!value,
+            })}
+            inputProps={{
+              className:
+                'PrivateSwitchBase-input-4 MuiSwitch-input cypress-boolean-input',
+            }}
+            value={!!value}
+            disabled={element.readOnly}
+            onChange={(e) => {
+              setIsDirty()
+              onChange(element, e.target.checked || undefined)
```

Comment:
I dont think is right. This will prevent the PDF and such from showing "No". We should be setting `false` on the submission.

Reply:
sorry, yep changed this just before we spoke about it and forgot to change back